### PR TITLE
Tr 2255 Updated fields that always return "Customs_aggr"

### PR DESCRIPTION
**Pull request by alexnorth**

### CODE REVIEW

File: index.md [hunk 1]
```diff
@@ -425,8 +426,8 @@ Taulukossa on kuvattu sanoman tietueiden käyttö.
 |Nimi|Tyyppi|Käytössä|Kuvaus|
 |:---|:---|:---|:---|
 |InformationRequestOpeningV01| | | |
-|&nbsp;&nbsp;&nbsp;&nbsp;InvstgtnId|Max35Text|Kyllä|Tutkinnan case id|
-|&nbsp;&nbsp;&nbsp;&nbsp;LglMndtBsis|LegalMandate1|Kyllä|Lainsäädäntöperuste. Numeroarvo välillä 100..n. Ensimmäiset numerot kertovat viranomaisen ja kaksi viimeistä numeroa lainsäädäntöperusteen. Lainsäädäntöperusteita kootaan Finanssiala Ry:n kanssa sovitun mukaisesti viranomaisten yhteisesti ylläpitämään kooditaulukkoon. Taulukko toimitetaan tiedonhakujärjestelmän toteuttajille sopimusneuvottelujen yhteydessä (taulukko ei ole julkinen).|
+|&nbsp;&nbsp;&nbsp;&nbsp;InvstgtnId|Max35Text|Ei|Aina "Customs_aggr"|
```
Comment (ext-tomminivanaho) [inline index.md]: Pitäisikö tässä olla kuitenkin "Kyllä" tuossa käytössä sarakkeessa? Vertaa alempi YesNoIndicator.

File: examples/Query_example-Finnish_PIC.xml [hunk 2]
```diff
@@ -116,9 +116,9 @@
             </urn1:AppHdr>
             <urn2:Document>
                 <urn2:InfReqOpng>
-                    <urn2:InvstgtnId>ABC1234</urn2:InvstgtnId>
+                    <urn2:InvstgtnId>Customs_aggr</urn2:InvstgtnId>
```
Comment (ext-tomminivanaho) [inline examples/Query_example-Finnish_PIC.xml:119]: Should we also update the investigation start date to 2020-09-01 while we are updating these? Does it matter that the signature does not match now after these updates, or should we update that also?

File: index.md (hunk 1 above)
Comment (alexnorth) [reply index.md]: No, the actual content of these 2 fields are not forwarded to consumer and will always have the value "Customs_aggr", hence not used.

File: examples/Query_example-Finnish_PIC.xml (hunk 2 above)
Comment (alexnorth) [reply examples/Query_example-Finnish_PIC.xml:119]: Updated dates, left signatures - they are just examples. Can update later if really needed

File: index.md (hunk 1 above)
Comment (alexnorth) [reply index.md]: I got what you meant now, yes it should be - updated

**Review by ext-tomminivanaho — APPROVED** [review]

File: index.md [hunk 3]
```diff
@@ -63,6 +63,7 @@ Versio|Päivämäärä|Kuvaus
 2.0.3|24.11.2022|Päivitetty ohjetiedostoja.|
 2.0.4|12.12.2022|Päivitetty asianajajan asiakasvaratiliin liittyviä rajoituksia. Asianajajan asiakasvaratilejä ei palauteta InformationResponseSD1V01 supl.027.001.01 alisanomassa, jos haun tyyppi on henkilö- tai organisaatiohaku.|
 2.0.5|7.2.2023|Tarkennuksia lukuun 4.12: Validointivirhettä voi käyttää virheellisen investigation periodin tapauksessa. Vastaussanoman maksimikoko on 5 Mb. Luvussa 3.1 VRK korvattu DVV:llä.|
+2.0.6|15.2.2023|Päivitetty 'Käytössä' ja 'Kuvaus' taulukoissa 4.5 InformationRequestOpeningV01 (InvstgtnId, LglMndtBsis) ja 4.6 AuthorityInquirySet (OfficialId, OfficialSuperiorId). Päivitetty esimerkkitiedostoja.|
```
Comment (ext-tomminivanaho) [inline index.md:66]: Update this to the top, still has "Dokumentin versio 2.0.5".

**Review by ekaivoso — APPROVED** [review]

**Pull request merged**